### ISSUE TITLE
With Any CPU build always get System.BadImageFormatException.

### ISSUE THREAD

**Issue by DanielTipo** [tuespetre/TuesPechkin#80]
Tried with referencing w32 & w64 wkhtmltox and manual assembly loading with CreateInstance. I have to use Any CPU build.

**Comment by tuespetre** [comment]
Sorry for the delay; do you have any of the details or did you figure this out?

**Comment by DanielTipo** [comment]
No, I didn't figure out. You can try it yourself setting a tutorial project to Any CPU. It won't work for you too, I assume.

**Comment by tuespetre** [comment]
Oops, I was closing some older/abandoned issues yesterday. I didn't mean to close yours. I will get an example together -- any CPU does work, it's the same as anything except, well, it runs on any cpu. I will get a sample together. Please provide a sample of your code if you can (like you mentioned, where you use CreateInstance.)

**Comment by ChrisVinall** [comment]
I had this problem and it was that the AppPool on IIS was set to forbid 32 bit applications from running. I'm using the 32-bit package in an app that is otherwise 64 bit. Don't recall why.

**Comment by DanielTipo** [comment]
ChrisVinall How did you recover this problem? Even if i use 32 bit package, with 32 bit enabled app pool, i get exception 0x8007007E (Unable to load DLL 'wkhtmltox.dll')

**Comment by ChrisVinall** [comment]
Don't know then. I just changed the AppPool settings.
